restapi/src/main/java/io/stargate/web/docsapi/resources/error/ErrorHandler.java

```diff
@@ -48,6 +49,12 @@ public Response apply(Throwable throwable) {
                   "Role unauthorized for operation: " + throwable.getMessage(),
                   Response.Status.UNAUTHORIZED.getStatusCode()))
           .build();
+    } else if (throwable instanceof OverloadedException) {
```

Comment:
should this be handled for gRPC as well?

Reply:
This PR is about HTTP APIs.

But yes, gRPC should also handle `Overloaded` in a protocol-specific way (most likely an error status).